packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -590,6 +590,7 @@ describeCompat(
 				// so dev builds (i.e. -dev or -dev-rc) sort as before official internal releases.
 				const isCurrentApi =
 					apis.dataRuntime.version.includes("dev") ||
+					apis.dataRuntime.version.match(/2\.0\.0-\d+/) !== null ||
```

Comment:
I don't know how to get from the comment to what this new rough match is trying to do. Can we add an additional comment for this? Should it also only allow versions starting with that pattern? Why don't 2.0.x need to match?

Reply:
I added details to the PR description. I don't want to re-run CI just to add a comment since this is blocking 2.0 release.

Reply:
^makes sense; let's file a ticket separately to add that comment in later. I will track that task.